setup.py

```diff
@@ -1,43 +1,6 @@
-# -*- coding: utf-8 -*-
-import os
+# Configuration has moved to pyproject.toml
+# This file is maintained for backward compatibility
```

Comment:
What kinds of setup is this supporting? Why can't we remove this file entirely?

Reply:
You're probably right, I think we could remove the file entirely. 

Only some old versions of pip/setuptools/etc might not support `pyproject.toml` projects correctly, but on 3.8+ users are advised to upgrade.

This should not be an issue, as we distribute universal wheels: almost no users would package from the sdist.